contracts/foundry.toml

```diff
@@ -0,0 +1,8 @@
+[profile.default]
+optimizer = false
```

Comment:
Let's change the default profile to:

```
[profile.default]
src = 'src'
out = 'out'
fuzz_runs = 500
optimizer = true
optimizer_runs = 999999
```

Reply:
Can't set the optimizer to true, because msize is not compatible with Yul optimizer. Get this error while compiling - 

> Error (6553): The msize instruction cannot be used when the Yul optimizer is activated because it can change its semantics. Either disable the Yul optimizer or do not use the instruction.

Reply:
But on the other side, @0xOsiris, correct me if I am wrong, setting too many optimizer runs will increase the size of the bytecode, and, hence, make "deployment" more expensive? Ik its staticcall but it still gotta get the runtime bytecode no?

Reply:
Yeah you make a good point. Deployment cost is really the important metric to optimize for